docs/source/topologies_beaker.rst

```diff
@@ -0,0 +1,36 @@
+Beaker Topologies
+=================
+
+.. contents:: Topics
+
+.. _beaker_topologies:
+
+
+Beaker Server
+```````````````
+
+.. code-block:: yaml
+
+    ---
+    topology_name: beaker
+    resource_groups:
+      - resource_group_name: test1
+        res_group_type: beaker
+        job_group: ci-ops-central
+        recipesets:
+          - distro: RHEL-6.5
+            arch: x86_64
+            arches:
+              - X86_64
+              - X86_64
+            keyvalue:
+              - MEMORY>1000
+              - DISKSPACE>20000
+            hostrequire:
+              - arch=X86_64
+    count: 1
```

Comment:
Good question.  This example topology was submitted a while ago, and I have used it as-is in a number of my recent test runs.  I'll try indenting the **count** key and I'll report my results.